### PR TITLE
clean code(code style) #1

### PR DESCRIPTION
**Pull request by abdotalaat**
apply the following rule "JUnit tests should include assert() or fail() " based on codacy result

### CODE REVIEW

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/DataObjectConverterTest.java [hunk 1]
```diff
@@ -24,12 +24,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/DataObjectConverterTest.java]: We should also assert the parsedModel

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/CustomNamespaceAttributeConverterTest.java [hunk 2]
```diff
@@ -19,12 +19,14 @@
...
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
     validateModel(parsedModel);
     deployProcess(parsedModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/CustomNamespaceAttributeConverterTest.java]: Is the deployProcess() method doing something? if not we should start getting rid of dead code.

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/DataStoreConverterTest.java [hunk 3]
```diff
@@ -16,12 +16,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/DataStoreConverterTest.java]: same as before

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/FormPropertiesConverterTest.java [hunk 4]
```diff
@@ -22,12 +22,14 @@
...
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
   
   @Test 
   public void doubleConversionValidation() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
     bpmnModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/FormPropertiesConverterTest.java]: This should be validated with an assertion as well

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/MessageConverterTest.java [hunk 5]
```diff
@@ -12,12 +12,14 @@
...
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
   
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
     validateModel(parsedModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/MessageConverterTest.java]: same here

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/MessageFlowConverterTest.java [hunk 6]
```diff
@@ -14,12 +14,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/MessageFlowConverterTest.java]: same here

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/OtherToolImportConverterTest.java [hunk 7]
```diff
@@ -12,12 +12,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/OtherToolImportConverterTest.java]: same here

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/PoolsConverterTest.java [hunk 8]
```diff
@@ -17,12 +17,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/PoolsConverterTest.java]: same here

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/ServiceTaskConverterTest.java [hunk 9]
```diff
@@ -19,12 +19,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/ServiceTaskConverterTest.java]: same here

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SignalConverterTest.java [hunk 10]
```diff
@@ -13,12 +14,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SignalConverterTest.java]: same here

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SimpleConverterTest.java [hunk 11]
```diff
@@ -19,12 +19,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull("BPMN Model XML not found", bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SimpleConverterTest.java]: same here

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SubProcessConverterNoDITest.java [hunk 12]
```diff
@@ -20,12 +20,14 @@
...
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void convertModelToXML() throws Exception {
     BpmnModel bpmnModel = readXMLFile();
+    assertNotNull(bpmnModel);
     BpmnModel parsedModel = exportAndReadXMLFile(bpmnModel);
```
Comment (Salaboy) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SubProcessConverterNoDITest.java]: same here

File: activiti-engine/src/test/java/org/activiti/engine/test/api/event/JobEventsTest.java [hunk 13]
```diff
@@ -200,6 +200,7 @@ public void testRepetitionJobEntityEvents() throws Exception {
   @Deployment
   public void testJobCanceledEventOnBoundaryEvent() throws Exception {
     Clock testClock = new DefaultClockImpl();
+    assertNotNull(testClock);
```
Comment (Salaboy) [inline activiti-engine/src/test/java/org/activiti/engine/test/api/event/JobEventsTest.java]: This one doesn't make much sense.. we just created the instance.

File: activiti-json-converter/src/test/java/org/activiti/editor/language/PoolConverterTest.java [hunk 14]
```diff
@@ -15,12 +15,14 @@
...
     BpmnModel bpmnModel = readJsonFile();
+    assertNotNull(bpmnModel);
     validateModel(bpmnModel);
   }
 
   @Test
   public void doubleConversionValidation() throws Exception {
     BpmnModel bpmnModel = readJsonFile();
+    assertNotNull(bpmnModel);
     bpmnModel = convertToJsonAndBack(bpmnModel);
```
Comment (Salaboy) [inline activiti-json-converter/src/test/java/org/activiti/editor/language/PoolConverterTest.java]: same here

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/DataObjectConverterTest.java (hunk 1 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/DataObjectConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/CustomNamespaceAttributeConverterTest.java (hunk 2 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/CustomNamespaceAttributeConverterTest.java]: all code inside deployProcess() method is commented, so i will comment the call and keep it

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/DataStoreConverterTest.java (hunk 3 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/DataStoreConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/FormPropertiesConverterTest.java (hunk 4 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/FormPropertiesConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/MessageConverterTest.java (hunk 5 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/MessageConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/OtherToolImportConverterTest.java (hunk 7 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/OtherToolImportConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/PoolsConverterTest.java (hunk 8 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/PoolsConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/ServiceTaskConverterTest.java (hunk 9 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/ServiceTaskConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SignalConverterTest.java (hunk 10 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SignalConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SimpleConverterTest.java (hunk 11 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SimpleConverterTest.java]: done

File: activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SubProcessConverterNoDITest.java (hunk 12 above)
Comment (abdotalaat) [inline activiti-bpmn-converter/src/test/java/org/activiti/editor/language/xml/SubProcessConverterNoDITest.java]: done

File: activiti-engine/src/test/java/org/activiti/engine/test/api/event/JobEventsTest.java (hunk 13 above)
Comment (abdotalaat) [inline activiti-engine/src/test/java/org/activiti/engine/test/api/event/JobEventsTest.java]: done

File: activiti-json-converter/src/test/java/org/activiti/editor/language/PoolConverterTest.java (hunk 14 above)
Comment (abdotalaat) [inline activiti-json-converter/src/test/java/org/activiti/editor/language/PoolConverterTest.java]: done

**Comment by Salaboy** [comment]
@abdotalaat there seems to be issues with your changes. Can you update your PR so I can merge it?

**Comment by abdotalaat** [comment]
@Salaboy  i fixed the conflict files and also remove some unused code .

@Salaboy why codecov checks failing?

**Comment by Salaboy** [comment]
@abdotalaat I think that @erdemedeiros removed the validation for tests without assertions, causing your contributions to not be as meaningful as before, but you can ask him ( @erdemedeiros ) about how to keep improving those tests. Let's check with him tomorrow.

**Comment by abdotalaat** [comment]
@Salaboy  it's fine if validation removed, i can start directly with this issue "https://github.com/Activiti/Activiti/issues/1261".

**Comment by Salaboy** [comment]
@abdotalaat yes you can start with that new issue, but please have a conversation with @erdemedeiros to see how you can contribute to the test improvements as well.

**Comment by Salaboy** [comment]
@abdotalaat I will close this one for now..as the ones that you are providing now are much more useful

**Pull request closed without merging**